data/bash-completion/fwupdmgr.in

```diff
@@ -77,11 +77,8 @@ _show_modifiers()
 _show_device_ids()
 {
 	local description
-	OLDIFS=$IFS
-	IFS=$'\n'
-	description="$(command fwupdagent get-devices 2>/dev/null | sed -e 's,"Name" :,,; s,",,g; s,\,,,g; s,[[:space:]]\+,,' | command awk '!/DeviceId/ { line = $0 }; /DeviceId/ { print $3 " { "line" }"}')"
+	description="$(command fwupdmgr get-devices 2>/dev/null | grep 'Device ID' | rev | cut -d' ' -f 1 | rev)"
```

Comment:
Reading fwupdmgr output isn't a good sustainable idea with tree output. I think we need to use json mode if wanting to parse it

Reply:
(hint, add --json and then use jq)

Reply:
And that means adding a recommends on jq to the packaging too if we don't already have it.

Reply:
Besides the `dependencies.xml`, do we need jq in `debian/control.in`?

Reply:
If it's a Recommends, then it should just go into `debian/control.in` and in fedora's spec file.